### DISCUSSION TITLE
hourly solar and wind generation data

### DISCUSSION THREAD

**Discussion by aesharpe** [catalyst-cooperative/pudl#2310] [Help Me!] [answered]
Copied from an email I received:

> I have been looking for hourly solar and wind generation data at the Balancing Authority, or unit level from 2010 - present. I haven’t been able to find this data for years before 2018 (EIA has some BA level hourly only 7/2018 to present). I was wondering whether you might have any ideas of where I could look for granular, renewable generation data?

**Comment by TrentonBush** [comment, marked as answer]
I'm pretty certain public, unit-level, hourly generation does not exist for most renewable generators. From what I remember, certain institutions do release that kind of data on the BA level -- maybe check Bonneville Power, BC Hydro, CAISO

**Comment by zaneselvans** [comment]
Do most people who care about this just use production models like those produced by NREL, plus whatever public wind & solar facility information is available to estimate historical + future RE production?

**Reply by TrentonBush** [reply]
That's my understanding, yes. My experience with production models in industry is that they are not that much different in concept. They just use site-specific weather models and measurements to tune the same long term reference weather data, plus their greater knowledge of historical performance to apply more accurate fudge factors for things like performance degradation, maintenance outages, and balance-of-plant electrical losses.
